pkgs/tools/typesetting/xmlto/default.nix

```diff
@@ -3,9 +3,10 @@
 , bash, getopt, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  name = "xmlto-0.0.28";
+  pname = "xmlto";
+  version = "0.0.28";
   src = fetchurl {
-    url = "http://fedorahosted.org/releases/x/m/xmlto/${name}.tar.bz2";
```

Comment:
Seems like there is a handful of other packages referencing fedorahosted.org. Seeing as that was retired, those also need this fixed (eventually).